draft-ietf-taps-interface.md

```diff
@@ -887,12 +887,27 @@ over these options.
 Name:
 : multipath
 
-This property specifies whether an application considers it useful to
-transfer data across multiple paths between the same end hosts. Generally,
-in most cases, this will improve performance (e.g., achieve greater throughput).
-One possible side-effect is increased jitter, which may be problematic for
-delay-sensitive applications. The recommended default is to Ignore this option.
+Type:
+: Enumeration
+
+This property specifies whether an application wants to take advantage of
+transferring data across multiple paths between the same end hosts. Using
+multiple paths allows connections to migrate between interfaces as
+availability and performance properties change. Possible values are:
+
+Disabled:
+: The connection will not attempt using multiple paths once established
+
+Handover:
+: The connection should attempt to migrate between different paths upon interface availability changes
+
+Interactive:
+: The connection should attempt to use multiple paths in response to loss or delay upon individual paths
+
```

Comment:
This one is a bit fuzzy, particularly "delay". Does it mean "delay spikes" or the base delay is over a given delay target? But perhaps it is hard to be precise? I think this will vary from implementation to implementation.